sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md

```diff
@@ -80,6 +81,20 @@ Only in the unlikely and regretful event of a release happening with a bug.
 - Fix the bug, and bump version (X.Y.Z+1).
 - A PR should be made on `master` from patch to directly resolve this particular bug.
 
+## PR review guidelines
+
+- Authors/Reviewers can request help for review at nf-core Slack `#request-review` channel.
+- Anyone can merge after one positive review and with no obvious open questions and, merge ASAP so as not to leave approved PRs hanging.
+- Exception in merges to master for releases (two reviews).
+- Exception in pseudo PR review for first release (needs review from core or maintainer team).
+- If it's a major change then it's ok to request a second review - either the author or reviewer can do this.
+- PRs that have approving reviews but open questions can be merged by the author after addressing the questions to a common-sense level of satisfaction.
+- If the reviewer feels uneasy about this then they shouldn't leave an approval, but rather a request-changes.
+- We prefer to use merge commits to merge. This is to avoid merge conflicts when multiple people are pulling with overlapping feature branches.
+- We prefer verbose commit histories but easy merges.
+- Feature branches should be immediately deleted after merge. Ideally by selecting the new feature on GitHub repo settings
+- Note: squash merging commits in a PR and merging after is fine, that is up to the individuals.
```

Comment:
Squash merging -> squashing 

Maybe needs a bit more explanation and is a sub point of the merge commit one above